### PR TITLE
[MX-410] Feature screen

### PR DESCRIPTION
**Pull request by ds300**
The type of this PR is: **Feature**

https://artsyproduct.atlassian.net/browse/MX-410

### Description

This adds a new screen to the app for so-called 'Feature' pages. I copied the code from [force](https://github.com/artsy/force/tree/master/src/v2/Apps/Feature) and adapted it for eigen.

### Test Plan

The designs were copied from force too, and then I paired with @samchieng  on modifying the layout slightly for eigen

### Screenshots

image

image

image

image

image

image

#skip_new_tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Missing Test Files:

- `src/lib/Scenes/Feature/components/__tests__/FeatureFeaturedLink-tests.tsx`
- `src/lib/Scenes/Feature/components/__tests__/FeatureHeader-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by samchieng** [comment]
This is 🔥

File: src/lib/Scenes/Home/Home.tsx [hunk 1]
```diff
@@ -159,9 +159,15 @@ const Home = (props: Props) => {
             }}
             ListHeaderComponent={
               <Box mb={1} mt={2}>
-                <Flex alignItems="center">
-                  <ArtsyLogoIcon scale={0.75} />
-                </Flex>
+                <TouchableOpacity
+                  onPress={() => {
+                    SwitchBoard.presentNavigationViewController(navRef.current, "/feature/alserkal-art-week")
```
Comment (ashfurrow) [inline src/lib/Scenes/Home/Home.tsx:164]: Should this line be included, or was it for developing?

Comment (ds300) [reply src/lib/Scenes/Home/Home.tsx:164]: ah it was for developing, thanks! removed.

**Comment by ds300** [comment]
There's something weird going on with this PR. I've pushed to the branch but github isn't showing the commits here. They're visible at https://github.com/artsy/eigen/tree/feature-screen though 🤔 if this doesn't resolve itself I'll try recreating the PR tomorrow.

**Comment by ds300** [comment]
looks like it's fixed itself now 🙌

File: Artsy/App/ARSwitchBoard.m [hunk 2]
```diff
@@ -267,6 +267,10 @@ - (void)updateRoutes
         [self.routes addRoute:@"/viewing-room/:id/artworks" handler:JLRouteParams {
             return [[ARViewingRoomArtworksComponentViewController alloc] initWithViewingRoomID:parameters[@"id"]];
         }];
+
+        [self.routes addRoute:@"/feature/:slug" handler:JLRouteParams {
+            return [[ARComponentViewController alloc] initWithEmission:nil moduleName:@"Feature" initialProperties:parameters ];
```
Comment (brainbicycle) [inline Artsy/App/ARSwitchBoard.m:272]:
```suggestion
            return [[ARComponentViewController alloc] initWithEmission:nil moduleName:@"Feature" initialProperties:parameters];
```
Extremely nitpicky / non-blocking

File: src/lib/Scenes/Feature/components/__tests__/FeatureMarkdown-tests.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Theme } from "@artsy/palette"
+import { extractText } from "lib/tests/extractText"
+import React from "react"
+import { create } from "react-test-renderer"
+import { FeatureMarkdown } from "../FeatureMarkdown"
+
+describe(FeatureMarkdown, () => {
+  it("renders markdown", () => {
```
Comment (brainbicycle) [inline src/lib/Scenes/Feature/components/__tests__/FeatureMarkdown-tests.tsx:8]: 🎉

**Pull request merged**